app/src/ui/history/commit-list.tsx

```diff
@@ -32,13 +33,15 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
 
     const gitHubUser =
       this.props.gitHubUsers.get(commit.author.email.toLowerCase()) || null
-    let avatarUser = null
-    if (gitHubUser) {
```

Comment:
In a few places we were only rendering the avatar when we had an associated GitHub user. By grabbing the Gravatar URL when we can't find these users, the experience with non-GitHub repositories should be much nicer...